Essentials/src/main/java/com/earth2me/essentials/Essentials.java

```diff
@@ -201,6 +201,10 @@ public void onEnable() {
             Console.setInstance(this);
 
             switch (VersionUtil.getServerSupportStatus()) {
+                case DANGEROUS: {
```

Comment:
Might be worth renaming this and/or the properties file to specify that this specifically covers Paper forks?